### PR TITLE
Oh Calculus, my dear old friend.

### PR DESCRIPTION
**Pull request by ilopezgp**
This PR computes the average over two percentiles of a Gaussian analytically, resulting in perfect precision and a 3 order of magnitude reduction in the cost of computation versus the current method. 

This should probably be ported over ClimaAtmos as well.

### CODE REVIEW

File: src/utility_functions.jl [hunk 1]
```diff
@@ -22,6 +21,15 @@ function percentile_bounds_mean_norm(
     return ∑samples / n_filtered_samples
 end
 
+"Compute average between two percentiles of a standard Gaussian"
```
Comment (charleskawczynski) [inline src/utility_functions.jl:24]:
```suggestion
"""Compute average between two percentiles of a standard Gaussian"""
```

File: src/utility_functions.jl [hunk 2]
```diff
@@ -1,5 +1,4 @@
-# compute the mean of the values between two percentiles (0 to 1) for a standard normal distribution
-# this gives the surface scalar coefficients for 1 to n-1 updrafts when using n updrafts
+"Compute the mean of the values between two percentiles (0 to 1) by sampling"
```
Comment (charleskawczynski) [inline src/utility_functions.jl:1]:
```suggestion
"""Compute the mean of the values between two percentiles (0 to 1) by sampling"""
```

**Review by charleskawczynski — APPROVED** [review]
Love it. Can we clean up the parameters that we no longer need? (`n_samples` and `set_src_seed`).

Overall, approve this ∫

**Comment by ilopezgp** [comment]
> Love it. Can we clean up the parameters that we no longer need? (`n_samples` and `set_src_seed`).
> 
> Overall, approve this ∫

I can do that. Should I leave the former method, or remove it as well?

**Comment by charleskawczynski** [comment]
I’m fine with removing it!

**Pull request merged**